test/toolchain/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("//rust:toolchain.bzl", "toolchain_tool")
+
+toolchain_tool.cargo(
```

Comment:
Wdyt about putting these targets somewhere in the rules_rust, so users don't have to put them into their BUILD files, but can use `@rules_rust//rust/toolchain:current_rustfmt_files` (name to be discussed)?

Reply:
I really like that. I'll refactor this to do that.

Reply:
I went with `current_exec_rustfmt_files` per the suggestion in https://github.com/bazelbuild/rules_rust/pull/792#discussion_r659588298

I'll resolve this to consolidate the conversation.